src/twisted/conch/ssh/transport.py

```diff
@@ -284,10 +280,6 @@ def _getSupportedCiphers():
         b"aes192-cbc",
         b"aes128-ctr",
         b"aes128-cbc",
-        b"cast128-ctr",
```

Comment:
I don't think that this change is ok.

The Twisted backward compatibility state that before removing something we first start by raising a dedicated deprecation warning, informing the users that this feature will be removed.
A new Twisted version should be released with this version.

Then, after 2 other releases the functionality can be removed.

More info here https://docs.twisted.org/en/stable/development/compatibility-policy.html

Reply:
I have refactored this to a deprecation warning with a test case.